src/FormMigration/FormMetaDecorator.php

```diff
@@ -784,4 +785,71 @@ public function getFormFeaturedImage()
 
         return $featuredImage;
     }
+
+    /**
+     * @unreleased
+     */
+    public function isActiveCampaignEnabled(): bool
+    {
+        $isFormEnabled = give_is_setting_enabled($this->getMeta('activecampaign_per_form_options'), 'customized');
+
+        $isFormDisabled = give_is_setting_enabled($this->getMeta('activecampaign_per_form_options'), 'disabled');
+
+        $isGloballyEnabled = give_is_setting_enabled(give_get_option('give_activecampaign_globally_enabled'), 'on');
+
+        Log::error('isActiveCampaignEnabled', [
+            'return' => !($isFormDisabled || (!$isGloballyEnabled && !$isFormEnabled)),
+            '$isFormEnabled' =>$isFormEnabled,
+            '$isFormDisabled' => $isFormDisabled,
+            '$isGloballyEnabled' => $isGloballyEnabled,
+        ]);
```

Comment:
What's this for?

Reply:
I was testing the accuracy of the enabled method. It's no longer needed. a1b5fcc69322be0eb3bd94217e446887a4b928e3